### PR TITLE
Number formatter component

### PR DESCRIPTION
**Pull request by ehoogerbeets**

### CODE REVIEW

**Comment by gouniLee** [comment]
I'm trying to run the test through `npm test`, `npm start'  and 'npm build`
Here are messages that I've got, it looks like that I'm doing wrong. Did I miss any steps?
* npm start
```
node scripts/start.js 
Cannot read property 'entryOption' of undefined
```
* npm build
```
goun@goun-LGE:~/Source/opensource_iLib/react-ilib$ node scripts/build.js 
Creating an optimized production build...
ilib-webpack-plugin: not writing data: locale data is not dirty or locale data size is zero
ilib-webpack-plugin: not writing data: locale data is not dirty or locale data size is zero
ilib-webpack-plugin: not writing data: locale data is not dirty or locale data size is zero
Failed to compile.

./src/App.css
Module build failed: ModuleBuildError: Module build failed: TypeError: Path must be a string. Received undefined
```
* npm test 

```
.....Error: Uncaught [Error: Cannot use count prop on a Translate component without giving both a "one" and "other" Plural component in the children.]
```

File: test/components/testAddress.jsx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+ */
+
+import React from 'react';
+import enzyme, { mount, render } from 'enzyme';
+import PropTypes from 'prop-types';
+import Adapter from 'enzyme-adapter-react-16';
+
+import LocaleContext from '../../src/components/LocaleContext';
+import Address from '../../src/components/Address';
+import ilib from 'ilib-es6';
```
Comment (gouniLee) [inline test/components/testAddress.jsx]: It looks like there is nowhere to be used, isn't it ok to be deleted?

Comment (ehoogerbeets) [reply test/components/testAddress.jsx]: The Address component is not merged or finished yet, and these tests don't even test anything real, so this file doesn't matter at the moment.

**Comment by ehoogerbeets** [comment]
The website is not working yet, so `npm build` and `npm start` don't work. The tests all pass, as you can see from the green OK at the end of the output. I had a hard time getting it not to print out the exceptions, which are the expected ones that we are testing for.

**Comment by gouniLee** [comment]
oh, I see. I've checked the Travis result which was the same on my machine. Even some error message shows it all passed test case. and Not only `testAddress.jsx` all test file includes `import ilib from 'ilib-es6'` includes. includes `testNumFmt.jsx`  That's why I mentioned it. it could be dealt with at one time later.

**Pull request merged**